ops/generate-daos-subgraph.js

```diff
@@ -4,6 +4,7 @@ const { migrationFileLocation: defaultMigrationFileLocation, network, startBlock
 const {   subgraphLocation: defaultSubgraphLocation } = require('./graph-cli')
 const path = require("path");
 const currentDir = path.resolve(`${__dirname}`)
+const supportedSchemes = ['ContributionReward', 'GenericScheme', 'SchemeRegistrar']
```

Comment:
what is this ?

Reply:
To index properly the schemes which the subgraph support and implemented.

Reply:
why it is needed to set explicitly here. ? 
how about daoFactory ?

Reply:
How is that related to the DAOFactory? This is related to which schemes the subgraph has mappings for and is indexing with full support, namely, the GenericScheme, ContributionReward, and SchemeRegistrar. It is not related to Arc or the DAOFactory in any way.

Reply:
why it is needed to set explicitly here. ?

Reply:
So that we can know if we should generate entities for them or not.